README.md

```diff
@@ -1,25 +1,29 @@
 ![Just One Single History](/splash.png)
 
 Combine the advantages of a monorepo with those of multirepo setups by leveraging a
-blazingly-fast, incremental, and reversible implementation of git history filtering.
+fast, incremental, and reversible implementation of git history filtering.
 
 `josh-proxy` can be integrated with any git host:
 
 ```
-$ docker run -p 8000:8000 -e JOSH_REMOTE=https://github.com -v josh-vol:/data/git joshproject/josh-proxy:latest
+$ docker run \
+    -p 8000:8000 \
+    -e JOSH_REMOTE=https://github.com \
+    -v josh-vol:/data/git \
+    joshproject/josh-proxy:latest
 ```
 
-See [Container options](#container-options) for full list of environment variables.
+See [Container options](https://josh-project.github.io/josh/reference/container.html) for full list of environment variables.
 
 ## Use cases
 
-### Partial cloning
+### Partial cloning <a href="https://josh-project.dev/~/ui/browse?repo=josh.git&path=&filter=%3A%2Fdocs&rev=HEAD"><img src="https://img.shields.io/badge/try_it-josh--project.dev-black"/></a>
```

Comment:
maybe we should also update to the latest UI version because as-is it's a bit underwhelming (and it's not obvious that it's not just a subfolder but a real git repo)